pysatNASA/instruments/ses14_gold.py

```diff
@@ -116,14 +116,14 @@ def clean(self):
 # Set the list_files routine
 fname = ''.join(('gold_l2_nmax_{year:04d}_{day:03d}_v{version:02d}',
```

Comment:
While we're at it, would it be better to make fname tag/inst_id dependent here?

Reply:
Looking at the other data products we'll eventually add, it's just a matter of swapping out nmax for the new product name.  We could either generate a dict here, or use something like 
```
datestr = '{year:04d}_{day:03d}_v{version:02d}_r{revision:02d}_c{cycle:02d}'
fname = 'gold_l2_{tag}_{datestr:s}_.nc'
```
with `fname.format(tag=tag, datestr=datestr)` at appropriate places below.

Reply:
Yes, I did something similar in pysatMadrigal with file type.  It looks like your example code does the same thing, so go for it!

Reply:
EDIT: found a cleaner method and pushed.